packages/features/shell/Shell.tsx

```diff
@@ -189,6 +191,28 @@ function useRedirectToOnboardingIfNeeded() {
   };
 }
 
+function useRedirectTo2FAIfNeeded() {
```

Comment:
hook to redirect organiztion user to two-factor-auth web page whenever they are navigating within the app